fire/test_main.py

```diff
@@ -0,0 +1,21 @@
+"""
+Test using fire via python -m fire
+"""
+import os
+
+from fire import __main__
+from fire import testutils
+
+class TestAsMainModule(testutils.BaseTestCase):
+  def test_name_setting(self):
+    # confirm one of the usage lines has tempfile gettempdir
+    with self.assertOutputMatches('tempfile gettempdir'):
+      __main__.main(['__main__.py', 'tempfile'])
+  def test_arg_passing(self):
```

Comment:
nit: blank line before each "def" in the class, and two blank lines before the "class" line at line 9.